source/firstprogram-expanding-network.rst

```diff
@@ -0,0 +1,341 @@
+
+Expanding the Network
+=====================
+
+Ealier we had one location which was the ``House``. In this section we increase the locations to ``House``,  ``Office``, and ``School``. Every person has a unique house and either a Office or a School and this categorized on the basis of age. 

```

Comment:
Don't capitalise O and S in office and school respectively, since they aren't classes.

Reply:
You mean in the last line right ? I have defined Office and School as separate classes.

Reply:
Yes, but in this line you aren't talking about the classes, you're making a more general statement about the kind of connections people have. Make this change.